cloud_service_settings.ini

```diff
@@ -4,10 +4,20 @@
 tts_service = google
 
 
+	# Which translation service will you use? DeepL is slower but more accurate
+	# Possble Values: google / deepl
+	# Remember, changing this service will require you to change the target language codes in the batch.ini file
+translate_service = google
```

Comment:
@alani1 I think this is what you're looking for, if set to DeepL, and assuming either Thio, me, or someone else implements the fallback, it would only translate using Google if DeepL is unavailable.

Reply:
Also, I was thinking of adding an option to stop translation, so all translations can be manually reviewed if needed, I think that would also help your case!